test/test_core.py

```diff
@@ -3747,6 +3747,17 @@ def test_dlfcn_asyncify(self):
       '''
     self.do_run(src, 'before sleep\nafter sleep\n42\n')
 
+  @requires_jspi
+  @needs_dylink
+  def test_jspi_dylink(self):
```

Comment:
Can you call this `test_dlfcn_jspi` maybe?

Reply:
Ideally it would be nice to have a better description but it's hard to come up with anything concise enough to be a function name.